crates/accelerate/src/target_transpiler/mod.rs

```diff
@@ -1076,24 +1091,21 @@ impl Target {
     /// Gets all the operation names that use these qargs. Rust native equivalent of ``BaseTarget.operation_names_for_qargs()``
     pub fn operation_names_for_qargs(
         &self,
-        qargs: Option<&Qargs>,
+        qargs: &[PhysicalQubit],
     ) -> Result<HashSet<&str>, TargetKeyError> {
         // When num_qubits == 0 we return globally defined operators
         let mut res: HashSet<&str> = HashSet::default();
         let mut qargs = qargs;
         if self.num_qubits.unwrap_or_default() == 0 || self.num_qubits.is_none() {
-            qargs = None;
+            qargs = [].as_slice();
```

Comment:
Does `&[]` not work?

Reply:
For this case it should, there are just some cases in which it doesn't work as easily.